data/856/884/75/85688475.geojson

```diff
@@ -667,6 +667,9 @@
     ],
     "wof:breaches":[],
     "wof:concordances":{
+        "digitalenvoy:region_code":[
+            51
```

Comment:
Do we know what these number are sourced from? They don't look like FIPS codes.

Reply:
I think these values were coded alphabetically.. 1-51, starting with Alabama, ending with Wyoming (and including Washington DC).

Reply:
Yeah, that was my guess, too. Since they drop the leading 0 we should match their oddness.

Reply:
Yeah, just confirming that there are no `0` prefix values `region_code` (or `metro_code` values). I'll keep these as-is, but change to string values.